ocaml/tests/test_xapi_db_upgrade.ml

```diff
@@ -42,88 +44,112 @@ let upgrade_vm_memory_for_dmc () =
   Db.VM.set_memory_dynamic_max ~__context ~self ~value:3L;
   Db.VM.set_memory_static_max ~__context ~self ~value:4L;
   (* Apply the upgrade rule *)
-  upgrade_vm_memory_for_dmc.fn ~__context;
+  X.upgrade_vm_memory_for_dmc.fn ~__context;
+
   let r = Db.VM.get_record ~__context ~self in
-  assert_equal ~msg:"upgrade_vm_memory_for_dmc: memory_dynamic_max <> memory_static_max"
-    r.API.vM_memory_dynamic_max r.API.vM_memory_static_max;
-  assert_equal ~msg:"upgrade_vm_memory_for_dmc: memory_target <> memory_static_max"
-    r.API.vM_memory_target r.API.vM_memory_static_max;
-  assert_equal ~msg:"upgrade_vm_memory_for_dmc: memory_dynamic_min <> memory_static_max"
-    r.API.vM_memory_dynamic_min r.API.vM_memory_static_max;
-  assert_bool "upgrade_vm_memory_for_dmc: memory_static_min > memory_static_max"
+  Alcotest.(check int64)
+    "upgrade_vm_memory_for_dmc: memory_dynamic_max <> memory_static_max"
+    r.API.vM_memory_dynamic_max
+    r.API.vM_memory_static_max;
+
+  Alcotest.(check int64)
+    "upgrade_vm_memory_for_dmc: memory_target <> memory_static_max"
+    r.API.vM_memory_target
+    r.API.vM_memory_static_max;
+
+  Alcotest.(check int64)
+    "upgrade_vm_memory_for_dmc: memory_dynamic_min <> memory_static_max"
+    r.API.vM_memory_dynamic_min
+    r.API.vM_memory_static_max;
+
+  Alcotest.(check bool)
+    "upgrade_vm_memory_for_dmc: memory_static_min > memory_static_max"
+    true
     (r.API.vM_memory_static_min <= r.API.vM_memory_static_max)
 
 let upgrade_bios () =
 
   let check inventory bios_strings =
-    Unixext.mkdir_safe "/var/tmp" 0o755;
-    Unixext.write_string_to_file "/var/tmp/.previousInventory" inventory;
-    let __context = make_test_database () in
-    upgrade_bios_strings.fn ~__context;
+    Stdext.Unixext.mkdir_safe "/var/tmp" 0o755;
+    Stdext.Unixext.write_string_to_file "/var/tmp/.previousInventory" inventory;
```

Comment:
It's better to reference the `Xapi_stdext_unix` module directly, that way we don't have to depend on the whole of stdext.